### ISSUE TITLE
There is 1 duplicated Feral File connection display when user connect to FF by Wallet Connect and Autonomy Connect

### ISSUE THREAD

**Issue by longbmk** [bitmark-inc/autonomy-apps#1968]
![image](https://user-images.githubusercontent.com/109057362/205793146-69a9c85e-1ea8-4f03-bb71-7e77e21acf62.png)

**Comment by hvthhien** [comment]
Technically, that's 2 different connections, so it is expected to show 2 rows. If we want to make user less confuse, we should have a new design (maybe divide into 2 sections for Au connect and Wc connect)

**Comment by longbmk** [comment]
@Anais-bitmark Please kindly help us design for it. Thank you

**Comment by longbmk** [comment]
@bm-kiennguyen Can we return the text "Feral File - Wallet Connect" and "Feral File - Autonomy connect" for 2 type of the connection? CC @hvthhien @Anais-bitmark

**Comment by Anais-bitmark** [comment]
@longbmk 

[Figma](https://www.figma.com/file/oz8vVT5wNpY3urnGoqVrnq/Au-bug_1968%3A-duplicated-FF-connection-display-when-user-connect-to-FF-by-WC-and-Au-Connec?node-id=0%3A1&t=dhJW9kxleHdWgtOW-1)

@hvthhien for Tezos, connections can happen through Beacon Standard or Autonomy Connect right?

<img width="947" alt="Capture d’écran 2023-01-06 à 12 27 19" src="https://user-images.githubusercontent.com/61187455/211012573-6b928641-e206-422f-a7b0-f5208a816fa7.png">

**Comment by longbmk** [comment]
@Anais-bitmark Thank you for your design. For the design for Tezos, please use "Feral File - Beacon Connect" or "Feral File - Beacon Wallet". CC @hvthhien

**Comment by anhnguyenbitmark** [comment]
@Anais-bitmark I don't think we should provide the suffix Wallet Connect and Autonomy Connect. They are protocol names, which should be confusing when the users see them.

Instead of showing this, can we prevent users from connecting to the Autonomy app (from Feral File) with WalletConnect? Since Feral File has Autonomy Connect and Other wallets at the same level, I would think we should not let the user use Wallet Connect in the "Other wallets" option. Unfortunately, we couldn't remove the Autonomy option in the Other wallets on the Feral File website.

**Comment by Anais-bitmark** [comment]
@anhnguyenbitmark it would be ideal but how so if we can't remove the Autonomy from the list of "Other Wallets"?
It will never work? or you didn't manage yet to remove it?

**Comment by Anais-bitmark** [comment]
[Figma](https://www.figma.com/file/oz8vVT5wNpY3urnGoqVrnq/Au-bug_1968%3A-duplicated-FF-connection-display-when-user-connect-to-FF-by-WC-and-Au-Connec?node-id=0%3A1&t=5dkBaYXXIxVsyxTu-1)

<img width="1332" alt="Capture d’écran 2023-02-22 à 15 57 53" src="https://user-images.githubusercontent.com/61187455/220680855-1a40f2cc-a8ed-42da-afef-9a2ceae5341c.png">

**Comment by anhnguyenbitmark** [comment]
@Anais-bitmark  It doesn't work. We cannot remove the wallet in the Beacon dialog.

**Comment by ppupha** [comment]
@bencebitmark please help to finalise the UI.

**Comment by bencebitmark** [comment]
@ppupha not sure if i understand this story correctly, but i think the issue doesn't happen anymore:
• in Autonony, there's only one Feral File connect listed for both the Tezos and Ethereum addresses
• Wallet Connect or Beacon modal doesn't list Autonomy twice

**Comment by ppupha** [comment]
@bencebitmark I check and it still happends. It's show 2 Feral File connection when we connect to FF using Autonomy Connect and Wallet Connect.
![1696493779031](https://github.com/bitmark-inc/autonomy-apps/assets/47568867/6fa17346-4bfd-4075-ba3f-bbd4e1aee4ff)

**Comment by bencebitmark** [comment]
oh okay, i managed to replicate it. just to clarify i understand correctly when this happens:
1. i connect to Feral File using Wallet Connect.
2. i then log out, or open a different browser. i use Autonomy Connect at Feral File.
3. two connections show up for Feral File on the same addresses.

is this correct?

**Comment by bencebitmark** [comment]
is it possible to combine the WalletConnect and Autonomy Connect items to just one? so when you disconnect, you disconnect both.

but tbf, i'm not sure if we're looking at this on the right level. this issue is unique to Feral File, and can't happen on any other dapp? is there value in us showing both WalletConnect and Autonomy Connect on Feral File?

**Comment by ppupha** [comment]
> Technically, that's 2 different connections, so it is expected to show 2 rows. If we want to make user less confuse, we should have a new design (maybe divide into 2 sections for Au connect and Wc connect)

@bencebitmark

**Comment by bencebitmark** [comment]
yes, it's two different connections, but i wouldn't change the entire screen for an issue that only occurs for a single dapp (Feral File). dividing it into Autonomy and Wallet Connect isn't the right approach here imo.

i still don't think we're addressing it at the right level: i'd like to address this at the root cause level instead of inventing a UI pattern that's not familiar to users.

i'd place this ticket in the backlog for now, as we're working out a solution that'd hide all of this info even more.

**Comment by ppupha** [comment]
@moskovich do you agree with this?

**Comment by moskovich** [comment]
I think it should only show one connection. Overwrite the last one.
